### DISCUSSION TITLE
[Feature] Allow filtering, searching for "Sharing" albums

### DISCUSSION THREAD

**Discussion by xamino** [immich-app/immich#12228] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Greetings! This might be user error, in which case I am happy to learn what my users and I are doing wrong, but here goes: we have a few hundred (but not thousand, yet) albums on our Immich instance that are shared amongst the family (joys of importing the family HDD :laughing: ). And while we all have access to them and can find any album we want by copious amounts of scrolling in the app or the "Sharing" view in the web UI, it doesn't make for a good user experience. However, in the web UI I can work around this by using the "Albums" view and toggling to "Shared" and then filtering down using the search or year etc.

It seems to me like the "Sharing" tab is not... feature complete, or maybe was superseded by the "Albums" tab. Either way, in the Immich app we don't have the full "Albums" view functionality via the "Library" tab, so all a user seems to be able to do is to go to "Sharing" in the app and start scrolling.

**Initial suggestion**
I'm not sure about the general goals and thoughts behind "Albums" vs "Sharing", but whatever the end goal, the functionality should be available in both views and the mobile app to search, filter, and sort or otherwise categorize many albums
* Mobile: allow searching, filtering, and sorting both the "Sharing" and "Library" views
* Web: allow searching, filtering, and sorting the "Sharing" view (assuming the functionality isn't being moved to the "Albums" view)

Minimal version: allow to search in the "Sharing" tab in both mobile and we views, and for the "Albums" tab in the mobile app.

And thank you to everyone involved in the development so far, Immich has gotten me into self-hosting and is a general treat of a software to use and run!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

## Maybe related
* https://github.com/immich-app/immich/discussions/3176 – that seems to be more about other functionality around sharing face detection etc
* https://github.com/immich-app/immich/discussions/9003 – seems to be for the mobile view and partner sharing only
* https://github.com/immich-app/immich/discussions/9140 – I think this would probably solve this issue, but it seems to be web only

**Comment by bo0tzz** [comment]
#2995

**Reply by xamino** [reply]
Thank you, and sorry for not finding it, I promise I tried and failed to search for duplicates before submitting. I'll go vote and comment there. :+1:
